src/app/shared/routing/group-router.ts

```diff
@@ -43,7 +43,7 @@ export class GroupRouter {
    */
   urlArray(route: GroupRoute, page?: 'edit'|'details'): any[] {
     // eslint-disable-next-line @typescript-eslint/no-unsafe-return
-    return urlArrayForGroup(route.id, route.path, page ?? this.currentGroupSubPage());
+    return urlArrayForGroup(route, page ?? this.currentGroupSubPage());
```

Comment:
`urlArray` could/should return a `UrlCommand`.
(may be for a future PR)

Reply:
Fixed